.gitlab-ci.yml

```diff
@@ -450,3 +450,76 @@ publish:mongodb-backup-runner:
   script:
     - *tag_n_push_to_gitlab_registry
 
+.template:trigger:acceptance-testing:
```

Comment:
Nit: put the template just before is going to be used, so between `trigger:backend-integration-testing:mender-qa` and `trigger:acceptance-testing:inventory-enterprise`

Reply:
sure. thanks.